README.md

```diff
@@ -342,6 +342,30 @@ While several Ruby books suggest the first style, the second is much more promin
 
 Refactoring is even better. It's worth looking hard at any code that explicitly checks types.
 
+- For single-line lambdas, `lambda` or `->` may be used.
+
+```ruby
+  scope :example, -> { where(:a => b) }
+
+  # Or
+
+  scope :example, lambda { where(:a => b) }
+```
+
+- For multi-line lambdas, prefer `lambda do ... end`
+
+```ruby
+  # bad
+  scope :long_example, lambda {
+    # ...
+  }
+
+  # good
+  scope :long_example, (lambda do
```

Comment:
Unfortunately, I think they are.

I've suggested preferring `do ... end` as that's what [The Style Guide](https://github.com/bbatsov/ruby-style-guide) recommends, although I do find this a little messy when the lambda is an argument.